python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -259,6 +275,72 @@ async def begin_standby(self, id_data):
         await super().begin_standby(id_data)
         self.cmd_standby.ack_in_progress(id_data, timeout=60)
 
+    def convert_int(self, value):
```

Comment:
The CSC class seems like an odd place to have this method. Maybe create a `utils` submodule?

Reply:
Agreed. There is only an agreed value for floats (which is math.nan). There is no default value for ints. I am open to suggestions :)

Reply:
probably the same as for float...